internal/cli/atlas/organizations/create.go

```diff
@@ -58,13 +58,13 @@ func CreateBuilder() *cobra.Command {
 	opts := new(CreateOpts)
 	cmd := &cobra.Command{
 		Use:   "create <name>",
-		Short: "Create an Ops Manager or Cloud Manager organization. This command is unavailable for Atlas.",
+		Short: "Create an organization and links it to the requesting API Key's organization.",
```

Comment:
to be clear this is what I mean
```suggestion
		Short: "Create an organization.",
		Long: "When authenticating using API keys the requesting keys most have cross organization billing enable and the resulting org will be linked to the paying org.",
```